src/graphengine.js

```diff
@@ -39,16 +37,30 @@ export class GraphEngine extends HTMLElement {
      * Parses data into an array while converting stripping the
      * measurement key
      * @returns {Array} of formatted objects
-*/
+    */
     parseData(data) {
         data = JSON.parse(data);
-        const structured = data.map(item => ({
-            time: item.time,
-            value: item.mean
-        }));
-        return structured;
-    }
+        let uniqueGroups = [];
```

Comment:
This `parseData` seems to be duplicated...but is `hedwig-main.umd.js` actually a generated file? If so, I recently discovered you can tell github it is generated and it will tend to hide it from PR diffs:

https://github.com/github/linguist#generated-code

Reply:
Yes, `hedwig-main.umd.js` is a generated file, it's checked into the ` docs/assets/scripts/` location so that the docs site can use it for displaying graphs, but... this does introduce a point that I've thought about we should maybe have a cloudbuild process that places this in a CDN bucket or similar so that we wouldn't have to check it in.

I follow on the `parseData` one function was formatting the data while the other was parsing the date. I just combined them in `./src/line-graph.js` as it may be a little too expensive to have separate loops for the entire dataset.